src/gui_common/fonts/BBCode-Image-VerticalCenterAlign-3.tres

```diff
@@ -0,0 +1,4 @@
+[gd_resource type="BitmapFont" format=2]
```

Comment:
This is needed to properly center align images vertically on rich text label: https://docs.godotengine.org/en/stable/tutorials/gui/bbcode_in_richtextlabel.html#image-vertical-offset